codebuild/bin/install_s2n_head.sh

```diff
@@ -26,18 +26,16 @@ fi
 
 BUILD_DIR=$1
 source codebuild/bin/jobs.sh
-source codebuild/bin/s2n_setup_env.sh
 cd "$BUILD_DIR"
 
 # Clone the most recent s2n commit
 git clone --depth=1 https://github.com/aws/s2n-tls s2n_head
-
-cmake s2n_head -Bbuild -DCMAKE_PREFIX_PATH="$LIBCRYPTO_ROOT"
```

Comment:
Curious, was cmake doing something incorrectly?

Reply:
I forgot why I switched this originally, but after reverting this cmake failed with boringssl and awslc, which will now be tested in V2. I put this back to make for now, and created #3274 and listed it in #3246.

Reply:
Was able to add it back in 71b5bff0856c3f0eb63ecefa35b5e46032fb4faf